### ISSUE TITLE
Another Batik dependency problem

### ISSUE THREAD

**Issue by willrogers** [ControlSystemStudio/cs-studio#2640]
I have built the latest master branch. I then run the following command:

```
./cs-studio -nosplash -application org.csstudio.opibuilder.converter.edl -h
```
I get a little pop-up saying 'An error has occurred. Please see the log file ...

I have attached the log file. Unfortunately it is another batik depdendency error and I cannot figure it out. Any ideas @shroffk?

I first saw this when re-enabling the unit tests on our Jenkins build.

[1586968616661.log](https://github.com/ControlSystemStudio/cs-studio/files/4482521/1586968616661.log)

**Comment by shroffk** [comment]
Hmm... No I have not come across this one as yet.
I don't use the converter much which is why we might have missed it.

**Comment by willrogers** [comment]
I think all you need to do to find this error is run the unit tests.

**Comment by willrogers** [comment]
I think this was resolved by #2647 .
